docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,130 @@
+**3. Delete the original configuration (optional)**
+
+Now that the configuration is saved as a fragment, you can delete the original config from your machine and _replace it with the fragment_.
+In this way, you can keep all your machines up to date whenever you change the fragment.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Add a fragment to multiple machines
+
+Now that you have a fragment, you can add it to as many machines as you'd like:
+
+{{< table >}}
+{{% tablestep link="/fleet/fragments/#add-a-fragment-to-a-machine" %}}
+{{<imgproc src="/get-started/try-viam/rover-resources/fragments/fragments_list.png" resize="700x" class="fill alignleft" style="max-width: 250px" declaredimensions=true alt="Add fragment">}}
+**1. Add the fragment to one machine**
+
+On your machine's **CONFIGURE** tab, click the **+** button and select **Insert fragment**.
+Search for your fragment and add it.
+
+Click **Save** in the upper right corner of the screen.
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/noun-repeat.svg" class="fill alignleft" style="max-width: 100px"  declaredimensions=true alt="Repeat">}}
+**2. Repeat for each machine**
+
+Repeat step 1 for each of the machines that you want to configure in the same way.
+
+If some of your machines have slight differences, you can still add the fragment and then add fragment mods in the next section.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Modify a fragment
+
+If your machines are similar but not identical, you can use a fragment with all of them, and then [edit it](/fleet/fragments/#modify-the-config-of-a-machine-that-uses-a-fragment) to customize select fields of the configuration without modifying the upstream fragment.
+
+{{% alert title="Note" color="note" %}}
+If you modify fields within a fragment, your modifications will act as overwrites.
+If you later update the upstream fragment, the updates to the overwritten fields will effectively not be updated.
+{{% /alert %}}
+
+{{< table >}}
+{{% tablestep link="/fleet/fragments/#modify-the-config-of-a-machine-that-uses-a-fragment" %}}
+{{<imgproc src="/use-cases/one-to-many/noun-edit.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Edit">}}
+**1. Edit your machine's config**
+
+Go to the **CONFIGURE** tab of the machine whose config you want to modify, and make your edits.
+
+Don't forget to **Save**.
+
+{{% /tablestep %}}
+{{% tablestep link="/fleet/fragments/#use-fragment_mods" %}}
+{{<imgproc src="/use-cases/one-to-many/noun-reset.svg" class="fill alignleft" style="max-width: 100px"  declaredimensions=true alt="Reset to fragment">}}
+**2. (Optional) Revert fragment modifications**
+
+When you make edits to fields in your machine's config, a top-level section called `"fragment_mods"` is added to your JSON config with [update operators](https://www.mongodb.com/docs/manual/reference/operator/update/positional/#---update-) that modify each field you edit.
```

Comment:
Please check Natalia's email on "overwrite fragments". I believe this has a UI now

Reply:
Yes, it does, and I have them use the UI in step 1 but didn't realize there was a revert button because I didn't see it yesterday :D Yay!

Reply:
(done)